charts/featureform/templates/ingress/issuer.yaml

```diff
@@ -5,19 +5,24 @@
 #  Copyright 2024 FeatureForm Inc.
 #
 
-{{ if .Values.publicCert }}
+{{ if .Values.cert.publicCert }}
 apiVersion: cert-manager.io/v1
 kind: Issuer
 metadata:
-  name: letsencrypt-prod
+  name: letsencrypt
 spec:
   acme:
-    server: https://acme-v02.api.letsencrypt.org/directory
+    server: {{
+              ternary
```

Comment:
q: how do these options work? is it just a set of cert servers?

Reply:
That's correct, it's a couple of hardcoded cert servers, with the staging one specified at https://letsencrypt.org/docs/staging-environment/. The production one is just the normal one we've been using before.

Reply:
Gotcha make sense.